user-storage-jpa/conf/quarkus.properties

```diff
@@ -0,0 +1,3 @@
+quarkus.datasource.user-store.db-kind=h2
```

Comment:
@pedroigor while I am reworking the quarkus properties behaviour i noticed that the standard transaction mode is "enabled" instead of "xa" as it is for us. I am not sure if we should advertise to set transaction mode to XA here, too ( `quarkus.datasource.user-store.transactions=xa` ), so wanted to ask if needed